src/screens/Home/Home.tsx

```diff
@@ -44,35 +43,31 @@ const Home = ({ navigation }: Props) => {
         type: LOAD_PRICING
       });
       setLoading(false)
-    } catch (error: any) {
+    } catch (error) {
       setLoading(false)
-      console.log(error.message)
     }
   }
 
   useEffect(() => {
     getPricing({ limit: 5 });
-    if (refreshCrypto) {
-      GetCrypto().then(value => SetCrryptos(value));
-    }
-  }, [refreshCrypto])
+    GetCrypto().then(value => SetCryptos(value));
+  }, [])
 
   return (
     <ScrollView>
       <Header />
       <ContainerTitle>
-        <TitleText>{crryptos.length > 0 ? "My List" : "No coins added yet"}</TitleText>
+        <TitleText>{cryptos.length > 0 ? "My List" : "No coins added yet"}</TitleText>
       </ContainerTitle>
       {loading ? <ActivityIndicator size='large' /> : (
         <ListContainer>
-          {crryptos && crryptos.map(item => <ListCrypto key={item.id} data={item} />)}
-          <MyButton onPress={() => { navigation.navigate("Crypto") }}>
+          {cryptos && cryptos.map(item => <ListCrypto key={item.id} data={item} />)}
+          <MyButton onPress={() => navigation.navigate.bind(null)("Crypto")}>
```

Comment:
You use a bind here but also let the arrow function, and It should be 
```
<MyButton onPress={navigation.navigate.bind(null, "Crypto")}>
```

Also is a good practice to store the routhes in a enum, like the theme, but for example: 

routes.tsx 
```
enum Routes {
   HOME: "Home",
   CRYPTO: "Crypto"
}

export default Routes
```